stdlib/_curses.pyi

```diff
@@ -539,3 +539,4 @@ if sys.version_info >= (3, 8):
         minor: int
         patch: int
     ncurses_version: _ncurses_version
+    window = _CursesWindow
```

Comment:
Can't find it in python docs:

```suggestion
    window = _CursesWindow  # undocumented
```

Reply:
Specifically, the docs say `window.something`, but seems like that `window` should be interpreted as just a variable name. Also, Python 3.7 docs say `window.something` too, even though it doesn't have `curses.window`.